test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderTests.cs

```diff
@@ -667,6 +667,106 @@ var csdl
         [Fact]
         public void ReadAnnotationWithoutSpecifiedValueAndUsePrimitiveDefaultValues()

```

Comment:
Rather than replicate the entire test for different cases of SRID and Variable, can we have a single [Theory] and pass in the different cases that then get concatenated into a single schema template?